flocker/control/_model.py

```diff
@@ -90,9 +90,7 @@ class IRestartPolicy(Interface):
 
 
 @implementer(IRestartPolicy)
-@attributes([], apply_immutable=True,
-            # https://github.com/hynek/characteristic/pull/22
-            apply_with_init=False)
+@attributes([Attribute("name", default_value=u"never")], apply_immutable=True)
```

Comment:
And if you ever do something like this where everything implementing an interface has a required attribute, you can and should document it:

```
from zope.interface import Interface, Attribute

class MyInterface(Interface):
    name = Attribute()
```